src/client/pydaos/pydaos_core.py

```diff
@@ -28,9 +28,9 @@
 
 # pylint: disable=import-error
 if sys.version_info < (3, 0):
-    import pydaos_shim_27 as pydaos_shim
+    import pydaos.pydaos_shim_27 as pydaos_shim
 else:
-    import pydaos_shim_3 as pydaos_shim
+    import pydaos.pydaos_shim_3 as pydaos_shim
```

Comment:
> * Isn't this file missing a `from pydaos import DAOS_MAGIC, PyDError` statement?

I'm not 100%, maybe it does.  What I've done in this PR is make it so that I can import the modules from both python2 and python3.  I've not tried using the module to do anything yet so I can't rule out further changes.

> * Line 40 needs one more space on its indent to align with the dictionary specification.

I don't understand what Line 40 is doing so I don't want to risk destabilising it by making whitespace changes.